versioned_docs/version-8.2/self-managed/platform-deployment/helm-kubernetes/upgrade.md

```diff
@@ -120,7 +120,77 @@ Camunda Platform v8.2 uses Keycloak v19 which depends on PostgreSQL v15. That is
 
 **Method 1: Upgrade the database schema to work with PostgreSQL v15**
 
-TBA
+The easiest way to upgrade major versions of postgresql is to start a port-forward,
+and then run pg_dump / pg_restore. The postgresql client versions are fairly flexible
+with different server versions, but for best results, we recommend using the newest
+client version.
+
+1. In one terminal, start a port-forward against the postgresql service
+
+```bash
+kubectl port-forward svc/<deployment-name>-postgresql 5432
```

Comment:
For `<deployment-name>`, we use `<RELEASE_NAME>` in all docs.

Reply:
:+1: Fixed